<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>total</class>
 <widget class="QDialog" name="total">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="handleWidth">
    <number>20</number>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_expense">
     <property name="text">
      <string>总  支  出：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_expense_num">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_income">
     <property name="text">
      <string>总  收  入：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_income_num">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_balanced">
     <property name="text">
      <string>净  支  出：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_balanced_num">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
